src/main/java/uk/gov/hmcts/juror/job/execution/jobs/dashboard/ams/AmsDashboardSendJob.java

```diff
@@ -3,20 +3,16 @@
 import lombok.Getter;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.juror.job.execution.jobs.LinearJob;
-import uk.gov.hmcts.juror.job.execution.service.contracts.SftpService;
 
 import java.util.Set;
 
 @Component
 @Getter
 public class AmsDashboardSendJob extends LinearJob {
```

Comment:
If we no longer upload the file this Job can be deleted

Reply:
Updated